### DISCUSSION TITLE
Source URL query strings - how?

### DISCUSSION THREAD

**Discussion by williamsdyyz** [imgproxy/imgproxy#1309] [Q&A] [answered]
Hi. The docs say that for plain urls "If the source URL contains a query string or @, you'll need to escape it."
My query strings are like `?w=50`
What does ecaping that mean exactly?  I tried `\?w\=50` and I tried url encoding like `%3Fw%3D50`
Neither worked & I can't find any examples in the docs or via Google :(

**Comment by DarthSim** [comment, marked as answer]
Hey @williamsdyyz!

Just replace `?` with `%3F` and `@` with `%40`.

**Reply by williamsdyyz** [reply]
Thanks!

You might want to be more specific in the docs - it's the only thing I had problems with. The rest is so easy especially with the docker image.
